packages/react-components/react-nav-preview/library/etc/react-nav-preview.api.md

```diff
@@ -217,7 +218,9 @@ export type NavDrawerHeaderSlots = DrawerHeaderSlots;
 export type NavDrawerHeaderState = DrawerHeaderState;
 
 // @public
-export type NavDrawerProps = ComponentProps<NavDrawerSlots> & DrawerProps & NavProps;
+export type NavDrawerProps = ComponentProps<NavDrawerSlots> & DrawerProps & NavProps & {
+    tabsterDomAttribute?: TabsterDOMAttribute;
```

Comment:
this should be `tabbable?: boolean`, right? I pulled it and built it locally and doing so did update this file to match the NavDrawer.types.ts` file, but I have no idea why the PR build is passing.

Reply:
totally should be - updated!